content/platforms/kubernetes/reference/GESHER.md

```diff
@@ -0,0 +1,255 @@
+---
```

Comment:
@alexmilowski @bmansheim This doc should move to Tasks section IMO. Too much for getting started and this is definitely not a reference doc. It needs some TLC and potentially breaking down but I suggest we do post it as is for now and take care of that later. Just not in the reference section.

Reply:
I agree that this is a task.

Reply:
I don't think we should include this at all in this PR. While we need to add Gesher, the current docs on GitHub would need to be refactored to be used directly. Also, this is not reference material - more like a how-to guide (a task)

Reply:
We should not include this in this PR. This document belongs in the tasks section and also needs some refactoring.